virtool/users/api.py

```diff
@@ -139,13 +138,11 @@ async def put(self, data: CreateFirstUserRequest) -> Union[r201[User], r400, r40
             data.handle, data.password
         )
 
-        session_id, session, token = await get_data_from_req(
-            self.request
-        ).sessions.create_authenticated(
+        _, token = await get_data_from_req(self.request).sessions.create_authenticated(
             virtool.http.authentication.get_ip(self.request), user.id
         )
 
-        self.request["client"].authorize(session, is_api=False)
+        session_id = self.request.cookies.get("session_id")
```

Comment:
You should get the session_id from the session returned when calling `created_authenticated`. As written I think this will keep the session_id as the old id from whatever session existed before. Would be a good idea to manually test to make sure the initial sign on flow works as expected. The latest dev version has a new parameter to allow spinning up of a temporary virtool instance